op-node/rollup/finalized/finalized_test.go

```diff
@@ -0,0 +1,58 @@
+package finalized
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/testutils"
+)
+
+var exHash = common.Hash{0x01}
```

Comment:
Is this supposed to mean "example Hash", or does `ex` stand for something else?

Reply:
Yes, it was used in the test for `ConfDepth` before and available in the module and I just added it back during the rebase. Happy to change it to a better name.